src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.ts

```diff
@@ -160,27 +164,70 @@ export class EntityCountDashboardComponent
     const groupedByEntity =
       field.dataType === EntityDatatype.dataType ? field.additional : undefined;
     const groups = groupBy(entities, fieldName as keyof Entity);
-    const groupCounts = groups.map(([group, entities]) => {
+
+    let groupCounts = groups.map(([group, entities]) => {
       const label = extractHumanReadableLabel(group);
       return {
         label: label,
         value: entities.length,
-        id: group?.["id"] || label,
+        id: extractGroupId(group),
         groupedByEntity: groupedByEntity,
       };
     });
 
+    // Merge "" and undefined groups into a single group with label: undefined
+    const notDefinedGroups = groupCounts.filter(
+      (g) => g.id === "" || g.id === undefined,
+    );
+    if (notDefinedGroups.length > 1) {
+      const merged = {
+        label: undefined,
+        value: notDefinedGroups.reduce((sum, g) => sum + g.value, 0),
+        id: "",
+        groupedByEntity: notDefinedGroups[0].groupedByEntity,
+      };
+      groupCounts = [
+        merged,
+        ...groupCounts.filter((g) => g.id !== "" && g.id !== undefined),
+      ];
+    }
+
     if (field.dataType === "configurable-enum") {
       const enumValues = this.configurableEnum.getEnumValues(field.additional);
+      const validIds = new Set(enumValues.map((ev) => ev.id));
       const groupCountsMap = new Map(
         groupCounts.map((aggregate) => [aggregate.id, aggregate]),
       );
-      const groupCountSorted = enumValues
-        .map((enumValue) => groupCountsMap.get(enumValue.id))
+
+      // Mark and collect invalid options
+      const invalidOptions = groupCounts
+        .filter((g) => g.id && !validIds.has(g.id))
+        .map((g) => ({
+          ...g,
+          label: undefined,
+          isInvalidOption: true,
```

Comment:
now each different invalid option category is listed separately on the dashboard. Can you all combine them, so that there is only once a "[invalid option]" that summarizes them all?

<img width="383" height="360" alt="image" src="https://github.com/user-attachments/assets/821529ba-737b-44fb-905c-be0f01224f24" />